libr/cons/input.c

```diff
@@ -477,7 +477,10 @@ R_API int r_cons_readchar() {
 	SetConsoleMode (h, mode);
 #else
 	r_cons_set_raw (1);
-	if (read (0, buf, 1) == -1) {
+	r_cons_sleep_begin ();
+	ssize_t ret = read (0, buf, 1) == -1;
```

Comment:
this is bool, not ssize_t

Reply:
No, it's not: https://linux.die.net/man/2/read

Reply:
you are not assigning the result of read, but the result of ssize_t == -1 which is bool

Reply:
lol omg yes of course